src/Tribe/Views/V2/Customizer.php

```diff
@@ -32,6 +32,55 @@ public function filter_sections( array $sections, $customizer ) {
 		return $sections;
 	}
 
+	public function include_global_elements_settings( $section, $manager, $customizer ) {
```

Comment:
:no_entry_sign: Missing doc comment for function include_global_elements_settings()

<sub>hash: 8a8099424b9b81b9338f9163d337ac1b</sub>